packages/instantsearch.js/src/widgets/hits/hits.tsx

```diff
@@ -127,13 +127,28 @@ const renderer =
       />
     );
 
+    const bannerComponent: HitsUiComponentProps<Hit>['bannerComponent'] = ({
+      banner,
+      ...rootProps
+    }) => (
+      <TemplateComponent
+        {...renderState.templateProps}
+        rootProps={rootProps}
+        templateKey="banner"
+        data={banner}
+        rootTagName="aside"
+      />
+    );
+
     render(
       <Hits
         hits={receivedHits}
         itemComponent={itemComponent}
         sendEvent={sendEvent}
         classNames={cssClasses}
         emptyComponent={emptyComponent}
+        banner={results?.renderingContent?.widgets?.banners?.[0]}
+        bannerComponent={templates.banner ? bannerComponent : undefined}
```

Comment:
I wanted to get a functional work in progress pushed up, but I'm sure this isn't the correct way to achieve our goal of making the banner template optional.